### PR TITLE
CCCD: Cannot remove first miscellaneous/travel fee from a claim

### PR DESCRIPTION
**Pull request by willmcb**
#### What

Allows users to remove misc fees / travel fees. Previously app would error if fields were cleared and a user tried to progress.

#### Ticket

[CBO-800](https://dsdmoj.atlassian.net/browse/CBO-800)

### CODE REVIEW

File: app/assets/javascripts/plugins/jquery.numbered.elements.js [hunk 1]
```diff
@@ -53,15 +53,22 @@
...
+          $(el).find(self.settings.action).text(removeText);
           $(el).find(self.settings.action).css('display', 'block');
         });
-      } else{
+      } else {
         items.each(function(idx, el) {
           $(el).find(self.settings.number).text('');
-          $(el).find(self.settings.action).css('display', 'none');
+          $(el).find(self.settings.action).text(clearText);
+          $(el).find(self.settings.action).css('display', 'block');
```
Comment (naseberry) [inline app/assets/javascripts/plugins/jquery.numbered.elements.js]: You could simplify lines 70 - 71 by method chaining:
```
$(el).find(self.settings.action)
  .text(clearText)
  .css('display', 'block');
```

File: features/claims/advocate/scheme_eleven/misc_fee_removal.feature [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    When I click "Continue" in the claim form
+    Then I should be in the 'Travel expenses' form page 
+
+    When I click the link 'Back' 
+    Then I should be in the 'Miscellaneous fees' form page
+    And the last 'miscellaneous' fee rate should be populated with '131.00'
+
+    When I click last remove link
+    Then I should not see 'Remove'
```
Comment (colinbruce) [inline features/claims/advocate/scheme_eleven/misc_fee_removal.feature]: Missing newline

**Review by colinbruce — APPROVED** [review]
1 slight fix needed but otherwise
LGTM 👍

File: features/claims/advocate/scheme_eleven/misc_fee_removal.feature (hunk 2 above)
Comment (willmcb) [reply features/claims/advocate/scheme_eleven/misc_fee_removal.feature]: 👍

**Pull request merged**